config/routes.rb

```diff
@@ -134,6 +134,11 @@
     end
     resources :team_memberships, only: [:destroy]
 
+    resources :invitations, only: [] do
```

Comment:
As per #67, I'm not sure of why we have this as a separate routeset to the normal `invitations` one. What's your logic?

Reply:
I've left my explanation in the other PR.